cclient/api/core/get_transactions_to_approve.h

```diff
@@ -17,20 +26,19 @@ extern "C" {
 #endif
 
 /**
- * Tip selection which returns trunkTransaction and branchTransaction. The input
- * value is depth, which basically determines how many bundles to go back to for
- * finding the transactions to approve. The higher your depth value, the more
- * "babysitting" you do for the network (as you have to confirm more
- * transactions).
- *
- * https://iota.readme.io/reference#gettransactionstoapprove
+ * @brief Does the <b>tip selection</b>
  *
- * @param service IRI node end point.
- * @param req Request containing number of bundles to go back to determine
- * the transactions for approval.
- * @param res Response containing transactions to approve (branch + trunk)
+ * Tip selection which returns <b>trunk</b> and <b>branch</b> transaction. The input value <b>depth</b>
```

Comment:
Suggestion - 
Tip selection which returns two tips that can be later used as trunk and branch